servers/src/grin/server.rs

```diff
@@ -361,7 +361,6 @@ impl Server {
 
 	/// Start a minimal "stratum" mining service on a separate thread
 	pub fn start_stratum_server(&self, config: StratumServerConfig) {
-		let edge_bits = global::min_edge_bits();
```

Comment:
I'd rather keep something like `let edge_bits = global::min_edge_bits() +1;` than having an hardcoded value later. What do you think?

Reply:
Sure, good suggestion.